zulipterminal/ui_tools/boxes.py

```diff
@@ -500,6 +501,16 @@ def footer_notify_invalid_recipient(self, error_msg: str) -> None:
         ]
         self.view.set_footer_text(footer_message, 3)
 
+    def loosing_focus_from_stream_box(self, key: str) -> bool:
```

Comment:
There a typo in the method name and in the commit description.

Reply:
I can see a typo in the commit description - "diables" in the first line?
But, I can't figure out the typo in the function name. Could you please clarify?

Reply:
Yeah, re the latter, it's lose vs loose.